packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -132,49 +129,49 @@ export class ExpressApp {
           })
           .end();
       }
-    }
+    };
 
-    function logDeprecated(req) {
+    const logDeprecated = (req) => {
       log.warn(
         'DEPRECATED',
         req.method,
         req.url,
         '(' + req.header('x-client-version') + ')'
       );
-    }
+    };
 
-    function getCredentials(req) {
```

Comment:
I don't know that this function should be converted to an arrow function if it's not using `this`

Reply:
It is the same, in this case I used arrow functions to declare all the functions with the same notation